lib/utils/utils.js

```diff
@@ -237,6 +237,16 @@ async function findThemes (appSettings) {
   return filteredFolders
 }
 
+function getBlacklistedExtensions () {
+  const csvList = process.env.IGNORE_EXT_CONFIG_FOR
+  if (csvList) {
```

Comment:
I'd prefer the early return here, if (!csvList) return []. What do you think?

Reply:
done